src/oqsconfig.h.cmake

```diff
@@ -201,6 +201,17 @@
 #cmakedefine OQS_ENABLE_SIG_mayo_5_avx2 1
 ///// OQS_COPY_FROM_UPSTREAM_FRAGMENT_ADD_ALG_ENABLE_DEFINES_END
 
+///// OQS_COPY_FROM_LIBJADE_FRAGMENT_ADD_ALG_ENABLE_DEFINES_START
+
+#cmakedefine01 OQS_LIBJADE_BUILD
```

Comment:
what's cmakedefine01?

Reply:
CMake 3.10 added support for both `cmakedefine` and `cmakedefine01` to `configure_file()`. Both are similar and from CMake 3.15 docs:

> Input file lines of the form #cmakedefine01 VAR will be replaced with either #define VAR 1 or #define VAR 0 similarly. 

It's syntactic sugar.